### ISSUE TITLE
Access an already created store?

### ISSUE THREAD

**Issue by joelmoss** [ctrlplusb/easy-peasy#312]
Think I must be missing something here, so I apologise if that is the case, but I am trying to access to store after it is created. Mainly to allow me to add/modify (`store.addModel`) the model dynamically.

I was trying to simply use the easy-peasy context (`easy-peasy/src/context`) with `useContext()`, but that doesn't seem to work...

```javascript
import StoreContext from 'easy-peasy/src/context'
// in my component...
const store = useContext(StoreContext)
```

The `store` var is always undefined 😖

Any help would be appreciated, as it is late here and I'm tired, so am probably missing something obvious.

thx

**Comment by waynevanson** [comment]
You should access it without context, as easy-peasy has handled this.

Did you read the tutorials? https://easy-peasy.now.sh/docs/tutorial/

Instead, you could use `useStoreState` to access the whole store.
Use `useStoreActions` to access actions

```js
import { useStoreState, useStoreActions} from 'easy-peasy'

const MyComponent = props => {
  const store = useStoreState(store => store) // store
  const addModel = useStoreAction(action => action.addModel) // your action
  return <div/>
}
```

**Comment by ctrlplusb** [comment]
Released in v3.1.0 

👍
